docs/pages/faqs.md

```diff
@@ -2,20 +2,6 @@
 
 This section of the documentation will cover common questions and encounters often asked by users and developers.
 
-## Tooling
```

Comment:
I am fine with keeping this here as well, in addition to a proactive note in the readme.